lib/repo_tools.pm

```diff
@@ -147,11 +147,19 @@ sub prepare_source_repo {
     }
     # source repository is disabled by default
     else {
+        # OSS_SOURCE is expected to be added
         if (script_run('zypper lr repo-source') != 0) {
             # re-add the source repo
             my $version = lc get_required_var('VERSION');
-            my $source_name = is_tumbleweed() ? $version : 'distribution/leap/' . $version;
-            zypper_call("ar -f http://download.opensuse.org/source/$source_name/repo/oss repo-source");
+            my $repourl;
+            # if REPO_OSS_SOURCE is defined - use it, if not fallback to download.opensuse.org
+            if (my $repo_basename = get_var("REPO_OSS_SOURCE")) {
```

Comment:
variable is not defined in variables.md

Reply:
All `REPO_` variables have same meaning, so they are define as `REPO_*`. I believe name is good enough to describe which exactly repo it is. WDYT?